src/NzbDrone.Core/Notifications/Plex/Server/PlexServerSettings.cs

```diff
@@ -43,6 +46,12 @@ public PlexServerSettings()
         [FieldDefinition(5, Label = "Update Library", Type = FieldType.Checkbox)]
         public bool UpdateLibrary { get; set; }
 
+        [FieldDefinition(6, Label = "Map Paths From", Type = FieldType.Textbox, Advanced = true, HelpText = "Sonarr Path, Used to modify series paths when Plex sees library path location differently from Sonarr")]
```

Comment:
`Sonarr path, used to modify series paths when Plex sees library path location differently from Sonarr`

or

`Sonarr path. Used to modify series paths when Plex sees library path location differently from Sonarr`

would be better. Similar issue with the `MapTo` path.